generate_example_plot.py

```diff
@@ -0,0 +1,23 @@
+import ClarkeErrorGrid as EGA
+import random
+import math
+import numpy as np
+
+def make_data():
+    #Generates Random CGM and and Reference data
+    #Creates an array of 300 data points and uses log scale to create a clustering 
```

Comment:
say _about_ 300 data points
since you might remove some if they are greater than 400